src/makeRobot.js

```diff
@@ -38,7 +38,58 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    wheels: wheels,
+    version: version,
```

Comment:
Если имя свойства === названию переменной можно использовать сокращенную запись:
```suggestion
    name,
    wheels,
    version,
```